src/docker/docker_command.rs

```diff
@@ -25,7 +25,7 @@ impl<'a> DockerCommand<'a> {
         cmd(
             "docker",
             self.arguments.iter().map(|argument| {
-                let argument_str: &'a str = argument.borrow();
+                let argument_str: &str = argument.borrow();
```

Comment:
The error message provides a good explanation for this change: https://crater-reports.s3.amazonaws.com/pr-98835/try%23cb1f7c96119295882e08b09b4bd01051669c071b/gh/jvff.dkr/log.txt